tests/ValueTests.hs

```diff
@@ -52,9 +53,9 @@ tests = testSpec "Value" $ do
     prop "Bool" $ prop_roundtripValue @Bool
     prop "Int32" $ prop_roundtripValue @Int32
     prop "Double" $ prop_roundtripValue @Double
-    prop "Text" $ forAll arbitraryText prop_roundtripValue
+    prop "Text" $ forAll arbitraryText (prop_roundtripValue @Text)
     prop "Lists" $ prop_roundtripValue @[Int32]
-    prop "Non-empty lists" $ forAll (arbitraryNonEmpty @Int32) prop_roundtripValue
+    prop "Non-empty lists" $ forAll (arbitraryNonEmpty @Int32) (prop_roundtripValue @(NonEmpty Int32))
```

Comment:
Ah no, leftover from when I was trying the associated type family for `FromValue`, removed